src/test/java/io/pinecone/PineconeIndexOperationsTest.java

```diff
@@ -140,6 +141,10 @@ public void testCreateIndexWithAllFields() throws IOException {
         assertEquals(createdIndex, expectedIndex);
         assertEquals(requestCaptor.getValue().method(), "POST");
         assertEquals(requestCaptor.getValue().url().toString(), "https://api.pinecone.io/indexes");
+
+        // Test for null CreateIndexRequest object
+        PineconeException thrown = assertThrows(PineconeException.class, () -> client.createIndex(null));
```

Comment:
Should it be a `PineconeValidationException` inside `assertThrows`? Also for this and the other `assertThrows` in tests could we use `PineconeValidationException` just to be explicit?

```suggestion
        PineconeValidationException thrown = assertThrows(PineconeValidationException.class, () -> client.createIndex(null));
```

Reply:
Ah yes good call! I must've missed updating this one!